_includes/accessibility-tests/jump-links.html

```diff
@@ -1,4 +1,4 @@
-## {{ component_name | capitalize }} accessibility checklist
+<h2>{{ component_name | capitalize }} accessibility checklist</h2>
```

Comment:
```suggestion
<!-- TODO: Reduce repetition in this template. -->
<h2>{{ component_name | capitalize }} accessibility checklist</h2>
```

Reply:
Update: I committed this TODO comment and opened issue #2599 for a follow-on effort.